src/Propulsion.SqlStreamStore/SqlStreamStoreSource.fs

```diff
@@ -3,17 +3,28 @@ namespace Propulsion.SqlStreamStore
 open System
 open SqlStreamStore
 
+type ReaderSpec =
```

Comment:
Hm not sure I love the existing ReaderSpec - that came about as there are so many knobs in the ES impl
Also, the global default block with overrides pattern is better replaced with individual commented optional args elsewhere 
though if you want to argue symmetry with .ES has value I guess thats fair...

Reply:
For me this many knobs without defaults was already too many. Symmetry with ES is a bonus.

Reply:
Yes, your call - however FsKafka has shown you can represent N options that have defaults as N optional args to a `Start` or `Create` that default.

The main thing you lose is the ability to couple all the settings in a record with minimal docs and pass them from one end of the app to another without thinking.

In general, this is a 'pattern' I'm looking to de-emphasize